src/components/LayerUI.tsx

```diff
@@ -130,6 +136,18 @@ const LayerUI = ({
 }: LayerUIProps) => {
   const device = useDevice();
 
+  const tunnels = useMemo(() => {
+    return {
+      mainMenuTunnel: tunnel(),
+      welcomeScreenMenuHintTunnel: tunnel(),
+      welcomeScreenToolbarHintTunnel: tunnel(),
+      welcomeScreenHelpHintTunnel: tunnel(),
+      welcomeScreenCenterTunnel: tunnel(),
+      footerCenterTunnel: tunnel(),
+      jotaiScope: Symbol(),
+    };
+  }, []);
```

Comment:
I've put it in `LayerUI`. We can surely factor it out into a hook, but it should live in the UI, not the App since we're trying to decouple the editor from UI.